homeassistant/components/calendar/__init__.py

```diff
@@ -137,6 +174,61 @@ def is_offset_reached(
     return start + offset_time <= dt.now(start.tzinfo)
 
 
+class CalendarEventDevice(Entity):
+    """Legacy API for calendar event entities."""
+
```

Comment:
We can log a warning like this:
https://github.com/home-assistant/core/blob/b022e08db9b722af39fe34ebd543d81be8e1744d/homeassistant/components/binary_sensor/__init__.py#L169-L175

Reply:
Thank you, had trouble finding a canonical example. I had trouble making `deprecated_class` work even copying an example.